### PR TITLE
Merge v0.3.0 into develop

### PR DESCRIPTION
**Pull request by ToniRamirezM**
Merges v0.3.0 into develop

### CODE REVIEW

**Review by tclemos — APPROVED** [review]

**Review by ARR552 — APPROVED** [review]
LGTM

**Review by Psykepro — APPROVED** [review]

File: config/environments/local/local.node.config.toml [hunk 1]
```diff
@@ -50,6 +51,16 @@ TrustedSequencerURL = "" # If it is empty or not specified, then the value is re
 WaitPeriodPoolIsEmpty = "1s"
 BlocksAmountForTxsToBeDeleted = 100
 FrequencyToCheckTxsForDelete = "12h"
+MaxTxsPerBatch = 300
```
Comment (Psykepro) [inline config/environments/local/local.node.config.toml]: The batch Constraints needs to be removed from here.

File: sequencer/config.go [hunk 2]
```diff
@@ -16,10 +16,41 @@ type Config struct {
 	// FrequencyToCheckTxsForDelete is frequency with which txs will be checked for deleting
 	FrequencyToCheckTxsForDelete types.Duration `mapstructure:"FrequencyToCheckTxsForDelete"`
 
+	// MaxTxsPerBatch is the maximum amount of transactions in the batch
```
Comment (Psykepro) [inline sequencer/config.go]: The batch Constraints needs to be removed from here.

**Review by Psykepro — CHANGES_REQUESTED** [review]
The batch Constraints need to be removed from the sequencer configs.

**Review by Psykepro — APPROVED** [review]

**Pull request merged**